util/testkit/handle.go

```diff
@@ -0,0 +1,19 @@
+package testkit
+
+import (
+	"testing"
+
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/sessionctx/stmtctx"
+	"github.com/pingcap/tidb/types"
+	"github.com/pingcap/tidb/util/codec"
+	"github.com/stretchr/testify/require"
+)
+
+func MustNewCommonHandle(t *testing.T, values ...interface{}) kv.Handle {
```

Comment:
You should add under "testkit" instead of "util/testkit". I will push a follow up commit.

Reply:
Done https://github.com/pingcap/tidb/pull/26629/commits/7c328357797fe42536c8bbc5ca526279a52c944f. Also be aware of License header which is required by APL 2.0.

Reply:
ok, thanks.